docusaurus/src/components/ConnectorRegistry.jsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import { useEffect, useState } from "react";
+
+const oss_registry_url =
```

Comment:
nit: we did have a json file to consume: https://connectors.airbyte.com/files/generated_reports/connector_registry_report.json